pages/list.html

```diff
@@ -0,0 +1,162 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>WALDO | List</title>
+    <link rel="stylesheet" href="../styles/main.css">
+</head>
+
+<body>
+    <nav class="navbar-content">
+        <img class="nav-icon" src="/assets/images/logo-cl.png" alt="">
+        <div class="nav-list">
+            <li><a href="../index.html">Home</a></li>
+            <li><a href="./install.html">Installation</a></li>
+            <li><a href="./documentation.html">Documentation</a></li>
+            <li><a href="https://github.com/pariyar07/WALDO">Contribute <i class="fab fa-github"></i></a></li>
+        </div>
+    </nav>
+    <aside id="aside-navbar">
+        <ul>
+            <li><a class="nav-link" href="./avatar.html">Avatar</a></li>
+            <li>
+                <a class="nav-link" href="./alert.html">Alert</a>
+            </li>
+            <li>
+                <a class="nav-link" href="./badge.html">Badge</a>
+            </li>
+            <li><a class="nav-link" href="./button.html">Button</a></li>
+            <li><a class="nav-link" href="./card.html">Card</a></li>
+            <li>
+                <a class="nav-link" href="./image.html">Image</a>
+            </li>
+            <li><a class="nav-link" href="./input.html">Input</a></li>
+            <li>
+                <a class="nav-link" href="./modal.html">Modal</a>
+            </li>
+            <li><a class="nav-link" href="./rating.html">Rating</a></li>
+            <li>
+                <a class="nav-link" href="./snackbar.html">Snackbar</a>
+            </li>
+            <li>
+                <a class="nav-link" href="./grid.html">Grid</a>
+            </li>
+            <li style="padding-left: 1rem; font-size: 1.5rem; background: #171717; border-radius: 8px;">
+                <a class="nav-link" href="#">List</a>
+            </li>
+        </ul>
+    </aside>
+    <main id="main-doc">
+        <div class="alert main-container">
+            <header>LIST</header>
```

Comment:
Header tag is not used for adding title or text

Reply:
Yup thanks. Corrected